requirements.txt

```diff
@@ -24,7 +24,7 @@ pytest-xdist # parallel execution
 pytest-cov
 pytest-timeout
 pytest-rerunfailures
-requests-mock
+pytest-httpx
```

Comment:
Don't forget to pin this and comment that there is an httpx conflict with QCS API client.

Reply:
Will do in the poetry PR 👍